doc/src/fpga_tools/fpgad/fpgad.md

```diff
@@ -5,15 +5,16 @@
 `fpgad [--socket=<sock>] [--null-bitstream=<file>]`
 
 ## DESCRIPTION ##
-```fpgad``` periodically monitors and reports the error status reflected in the device driver's error status sysfs files.
-```fpgad``` establishes the channel to communicate events to the Open Programmable Accelerator Engine (OPAE) application. 
-```fpgad``` programs a NULL bitstream in response to an AP6 (power) event. ```fpgad``` is only available on the Integrated FPGA
-Platform. You cannot run ```fpgad``` on the PCIe&reg; Accelerator Card (PAC).
+fpgad monitors the device sensors, checking for sensor values that are out of the prescribed range. 
 
-If your system does not support interrupts, you must run ```fpgad``` before the API calls `fpgaRegisterEvent` and
-`fpgaUnregisterEvent` can succeed.
+When any of the sensors is detected to be out of bounds, fpgad responds by removing the OPAE kernel driver for a configurable cool-down period.
```

Comment:
This statement is not true.

Reply:
When any of the sensors is detected to be out of bounds, fpgad will focus on keeping the server from rebooting (by masking AER) if/when the VC card is kicked off the PCIe bus by the BMC if a threshold is exceeded.